project/ProjectPlugin.scala

```diff
@@ -50,7 +50,6 @@ object ProjectPlugin extends AutoPlugin {
       addSbtPlugin(%%("sbt-scoverage", true)),
       addSbtPlugin(%%("sbt-scalajs", true)),
       addSbtPlugin(%%("sbt-header", "3.0.2", true)),
-      addSbtPlugin(%%("tut-plugin", true)),
```

Comment:
@juanpedromoreno I removed it since it's not used, not sure if this is the right strategy?

Reply:
I think it's fine 👍